src/homework01.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# coding: utf8
+
+"""
+EPAM python q4 homework 01
+Sergey Streltsov 2019-10-18
+"""
+
+import csv
+import doctest
+import glob
+import itertools
+from itertools import groupby
+import random
+import re
+import sys
+from time import localtime
+import unittest
+import xml.etree.ElementTree as etree
+
+# 1 line: Output
+print('Hello, world!')
+
+# 2 lines: Input, assignment
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+
+# 3 lines: For loop, built-in enumerate function, new style formatting
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print('iteration {iteration} is {name}'.format(iteration=i, name=name))
+
+# 4 lines: Fibonacci, tuple assignment
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+
+# 5 lines: Functions
+def greet(name):
+    print('Hello', name)
+
+
```

Comment:
why this function's calls are separated from this function with 2 lines and next task with one from calls?

Reply:
this recommendation comes from PyCharm, PEP8: expected 2 blank lines, found 1, i think this problem is due i join all tasks in one file